git.go

```diff
@@ -363,9 +364,27 @@ func (s *GitRepo) Ping() bool {
 	return err == nil
 }
 
+// EscapePathSeparator escapes the path separator by replacing it with several.
+// Note: this is harmless on Unix, and needed on Windows.
```

Comment:
You can detect OS with [build tags](https://godoc.org/go/build#hdr-Build_Constraints), or by checking [`runtime.GOOS`](https://godoc.org/runtime#GOOS) constant.

That way, you do only what's neccessary for each OS.